app/include/user_config.h

```diff
@@ -1,37 +1,206 @@
 #ifndef __USER_CONFIG_H__
 #define __USER_CONFIG_H__
 
```

Comment:
With all the relevant documentation now inside this file could you maybe add a note with a link to it to the first paragraph at https://nodemcu.readthedocs.io/en/latest/en/build/#build-options?

Reply:
Will do :smile: